go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	k8s.io/api v0.28.1
 	k8s.io/apimachinery v0.28.1
 	k8s.io/client-go v0.28.1
-	k8s.io/kubernetes v1.28.4
+	k8s.io/kubernetes v1.15.0-alpha.0
```

Comment:
That looks wrong.
Be careful about using the `-u` option on `go get`.  I generally don't add that option unless I explicitly want to do a general dependency update.